terminal/command_test.go

```diff
@@ -98,3 +99,8 @@ func TestExecuteFile(t *testing.T) {
 		t.Fatalf("Wrong counts break: %d trace: %d\n", breakCount, traceCount)
 	}
 }
+
+func TestIssue354(t *testing.T) {
+	//printStack([]api.Stackframe{ }, "")
```

Comment:
I actually want it to be there, I just commented it to check that it crashed on the second one too.